nuntium/views.py

```diff
@@ -85,6 +85,11 @@ def dispatch(self, request, *args, **kwargs):
 
     def get(self, *args, **kwargs):
         # Check that the form contains valid data
```

Comment:
You might want to move this comment down again, and add a comment about what this part is actually doing, but otherwise this all looks great!